RoddyCore/src/de/dkfz/roddy/knowledge/files/BaseFile.groovy

```diff
@@ -685,11 +686,15 @@ abstract class BaseFile<FS extends FileStageSettings> extends FileObject {
             }.flatten().join("\n\t")
             sb << "\nNote these may not have matched because they may be specific for a tool!"
 
-            throw new ConfigurationError(sb.toString(), baseFile.executionContext.configuration)
+            if (Roddy.getFeatureToggleValue(AvailableFeatureToggles.FailOnAutoFilenames))
+                throw new ConfigurationError(sb.toString(), baseFile.executionContext.configuration)
```

Comment:
See below. Also here a generic ConfigurationError is insufficient, if no message is provided about the reason (FailOnAutoFilenames).